### PR TITLE
run ant-design on windows [!depends on https://github.com/benjycui/bisheng/pull/62]

### PR DESCRIPTION
**Pull request by bchenSyd**
this PR depends on 
https://github.com/benjycui/bisheng/pull/62

please merge my PR after PR62 get merged into bisheng. thanks

### CODE REVIEW

**Comment by yesmeck** [comment]
`markdown-tree.debug` should not be committed?

**Comment by bochen2014** [comment]
1. markdown tree debug file removed
2. fix lint issue

you don't have to wait for bisheng's updated package. but it won't work on windows until bisheng is updated

File: package.json [hunk 1]
```diff
@@ -136,22 +138,19 @@
...
     "test-all": "./scripts/test-all.sh",
     "lint": "npm run tslint && npm run eslint && npm run demolint && npm run lesshint",
-    "tslint": "antd-tools run ts-lint && npm run compile && rm -rf lib",
+    "tslint": "antd-tools run ts-lint && npm run compile && rimraf lib",
     "eslint": "eslint tests site scripts components ./.eslintrc.js ./webpack.config.js --ext '.js,.jsx'",
-    "demolint": "RUN_ENV=DEMO eslint components/*/demo/*.md --ext '.md'",
+    "demolint": "cross-env RUN_ENV=DEMO eslint components/*/demo/*.md --ext '.md'",
     "lesshint": "lesshint components -r scripts/lesshint-report.js",
     "eslint-fix": "eslint --fix tests site scripts components ./.eslintrc.js ./webpack.config.js --ext '.js,.jsx' && eslint-tinker ./components/*/demo/*.md",
-
```
Comment (benjycui) [inline package.json]: These empty lines should be reserved.

**Comment by benjycui** [comment]
Empty lines in `package.json` can be removed.

We can use namespace to improve readability after this PR is merged.

```
lint
lint:ts
lint:es
lint:demo
```

**Comment by benjycui** [comment]
@RaoHai 这个改动可以体现在 Change Log，因为这意味着 Windows 的用户也可以向 ant-design 贡献了。

**Pull request merged**